clang-tools-extra/test/clang-tidy/checkers/readability/use-cpp-style-comments.cpp

```diff
@@ -0,0 +1,80 @@
+// RUN: %check_clang_tidy -std=c++11 %s readability-use-cpp-style-comments %t 
```

Comment:
Is c++11 mode necessary?
```suggestion
// RUN: %check_clang_tidy -std=c++98-or-later %s readability-use-cpp-style-comments %t 
```